### DISCUSSION TITLE
Is it possible to temporarily maximize window?

### DISCUSSION THREAD

**Discussion by fvacek** [helix-editor/helix#2251] [Q&A]
Like `Ctrl-b z` in tmux?

**Comment by workingj** [comment]
F11 on GNOME ?

**Reply by aral** [reply]
<kbd>F11</kbd> is to make a window fullscreen so we shouldn’t overload this. It could/should be used by terminal apps to go into fullscreen mode.

**Comment by sudormrfbin** [comment]
Maximizing splits is not possible yet, I'll make an issue from this discussion.
